packages/@lwc/compiler/src/index.ts

```diff
@@ -15,4 +15,4 @@ export {
     OutputConfig,
 } from './options';
 
-export const version = '__VERSION__';
+export const version = process.env.LWC_VERSION;
```

Comment:
We were already using `process.env.LWC_VERSION` elsewhere, so this is just being consistent.